flake.nix

```diff
@@ -0,0 +1,25 @@
+{
+  description = "Parse and compare all the package versions and all the ranges. From debian, npm, pypi, ruby and more. Process all the version range specs and expressions.";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+  };
+
+  outputs = { self, nixpkgs }: ({
+    overlays.default = import ./overlay.nix;
+  }) // (
+    let
+      supportedSystems = [ "x86_64-linux" "aarch64-linux" ];
```

Comment:
FWIW, this runs on all OS and arches.

Reply:
The point I was making is that the code is not architecture-specific at all, so IMHO we should not constraint supportedSystems? But I may not understand everything there.